_authors/arbel.md

```diff
@@ -1,3 +1,7 @@
 ---
 title: "Keren Arbel"
 ---
+Dr. Arbel has been immersed in Buddhist practice since 1997, practicing with variety of teachers in Asia and the West. She also holds a PhD in Buddhist Studies.
+
+She teaches workshops and retreats on mindfulness and insight meditation (vipassana), loving kindness  (metta) and Buddhist wisdom at the Israeli Insight Meditation Society [(Tovana)](https://www.tovana.org.il/en). She is a co-founder and a teacher at [(Nalanda)](https://www.nalanda.org.il/vision-en): an Israeli Buddhist institute dedicated to contemplative study of the Buddhist path. She also teaches in the MBSR teacher training program and the MBCT Facilitation Certificate at IDC Herzliya’s [Muda](https://muda.idc.ac.il/wp-content/uploads/2020/08/Mindfulness-Based-Social-Resilience-Platform-Muda-Aug2020-1.pdf) Institute for Mindfulness, Science and Society. For more than ten years she taught Buddhist philosophy and psychology as an adjunct professor in the Department of East Asian Studies at Tel Aviv University. Since 2017 she dedicates all her time to teaching the Dhamma.
+_~ Adapted from [Dr. Arbels website](http://kerenarbel.com/en/)_
```

Comment:
Please add a blank newline between these, otherwise it won't get a new paragraph. Refer to [this document](https://kramdown.gettalong.org/quickref.html) for any questions about markdown syntax